resident/resident-service/src/main/java/io/mosip/resident/validator/RequestValidator.java

```diff
@@ -1090,7 +1091,7 @@ public void validateDownloadCardRequest(MainRequestDTO<DownloadCardRequestDTO> d
 	}
 
 	private void validateIndividualIdV2(String individualId) {
-		if (individualId == null || StringUtils.isEmpty(individualId)) {
+		if (individualId == null || StringUtils.isEmpty(individualId) || isNumeric(individualId)) {
```

Comment:
We should not assume that individual ID is always numeric. If required use the UIN validator/VID validator/RID validator

Reply:
replace with rid validator.